client/quiz.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import { fetchJSON, postJSON } from "./http";
+import { useLoader } from "./useLoader";
+
+function ShowQuestion({ question, onReload }) {
+    async function handleAnswer(answer) {
+        const { id } = question;
+        postJSON("/quiz/answer", { id, answer });
+        onReload();
+    }
+    return (
+        <div>
+            <h2>{question.question}</h2>
+            {Object.keys(question.answers)
+                .filter((a) => question.answers[a])
+                .map((a) => (
+                    <div key={a}>
+                        <button onClick={() => handleAnswer(a)}>
+                            {question.answers[a]}
+                        </button>
+                    </div>
+                ))}
+        </div>
+    );
+}
+
```

Comment:
Det ser ut som om denne ikke er formatter med prettier. Det er en billig investering som reduserer sjansen for teite problemer